gui/layers/_visual_wrapper.py

```diff
@@ -84,6 +87,22 @@ def opacity(self, opacity):
 
         self._node.opacity = opacity
 
+    @property
+    def blending(self):
+        """string: Selects blending mode. Can be one of ('opaque',
```

Comment:
Typically, a pseudo-enum is represented as `{a, b, c}`. Would also prefer you to include a bullet-pointed list explaining what each does:
```python
"""{'opaque', 'translucent', 'additive'}: Blending mode...

    'opaque'
        Lorem ipsum...

    'translucent'
        Dolor sit...

    'additive'
        Amet...
"""
```

See the [`ndimage` docs](https://docs.scipy.org/doc/scipy/reference/generated/scipy.ndimage.maximum_filter1d.html#scipy.ndimage.maximum_filter1d) for an example.

Reply:
Yep will do! Thanks for the tip